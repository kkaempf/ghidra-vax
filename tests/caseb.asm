0x00000000: CASEB @#0x20040005, S^#0x1, S^#0x2
