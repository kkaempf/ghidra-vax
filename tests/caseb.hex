8f9f0500042001021000070019000000
