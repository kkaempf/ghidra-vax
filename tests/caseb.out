0x00000000: CASEB @#0x20040005, S^#0x1, S^#0x2
0x00000008: BSBB 0xa
